typings/index.d.ts

```diff
@@ -3255,6 +3257,14 @@ export interface CommandInteractionOption {
   message?: Message | APIMessage;
 }
 
+export interface CommandInteractionResolvedData {
+  users?: Collection<string, User>;
+  members?: Collection<string, GuildMember | APIInteractionDataResolvedGuildMember>;
+  roles?: Collection<string, Role | APIRole>;
+  channels?: Collection<string, GuildChannel | APIInteractionDataResolvedChannel>;
```

Comment:
Shouldn't `channels` here also allow `DMChannel`s?

Reply:
I don't think so - `interaction.options.getChannel()` is typed as GuildChannels. What channel can you select in a DM?

Reply:
Well, cough, sorry but that should also include Thread Channels at least, I'll test if you can select external channels in DMs

Reply:
Channels in DMs will error unless you select the current DM channel, which is currently possible

Reply:
TLDR: Channels can contain any channel type except Partial DM Channel from my testing. This should be updated to include those.